packages/rx_bloc_cli/mason_templates/bricks/lib_auth_matrix/__brick__/lib/lib_auth_matrix/views/auth_matrix_otp.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/widget_toolkit.dart';
+import 'package:widget_toolkit_otp/widget_toolkit_otp.dart';
+
+import '../../app_extensions.dart';
+import '../../feature_otp/services/otp_text_field_validator.dart';
+import '../../lib_router/blocs/router_bloc.dart';
+import '../bloc/pin_otp_bloc.dart';
+import '../models/auth_matrix_response.dart';
+import '../services/auth_matrix_service.dart';
+
+class AuthMatrixOtpPage extends StatelessWidget {
```

Comment:
Please, rename the file to `auth_matrix_otp_page.dart`. Also we currently have 2 identical OTP pages that are included in the generated project.